debian/aarch64/jessie/entry.sh

```diff
@@ -65,16 +71,20 @@ function init_non_systemd()
 	# trap the stop signal then send SIGTERM to user processes
 	trap signal_handler SIGRTMIN+3 SIGTERM
 
-	udevd & 
-	udevadm trigger &> /dev/null
-	
 	CMD=$(which "$1")
 	# echo error message, when executable file doesn't exist.
 	if [ $? == '0' ]; then
 		shift
-		"$CMD" "$@" &
-		pid=$!
-		wait $pid
+		if [ ! -z "$RESIN" ] && [ ! -z "$RESIN_DEVICE_UUID" ]; then
+			# run this on resin device only
+			start_udev
+			
+			"$CMD" "$@" &
+			pid=$!
+			wait $pid
+		else
+			exec "$CMD" "$@"
```

Comment:
@petrosagg we can't solve the interactive shell issue, if we do that since it's started as subshell. So I think it's better to note down RTMIN+3 stop signal in README so users will know and handle it manually.